packages/microcosm-www-next/src/stylesheets/components/_browser-graphic.scss

```diff
@@ -83,6 +84,14 @@ $left-padding-lg: rem(40);
 
   .graphic {
     background: linear-gradient(#5385e1 70%, #3a69c0 70%);
+
+    .section.-bg-6 &::after {
+      content: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAsAAAAVCAYAAACQcBTNAAAA0ElEQVQokZXRMUoDURAG4E+0EiVgCAQEK9ucIHZpPEIaLbxACq9g7UFEsAzkAIJWYiCk8gCBRCFYCJJnsW9VllVmp3nM8DE85odzDWqJQRS/4RXHUZwwRyuKE8bYjuKE6yY44aIJ/sBJFCcscBTFCc/YjeKEO2xFccLVf3iFF7znfoPhTuX/U/SwVsS/wR4O8/u9+VER90PuT/863RMOcj/K+LYO36Pzq+/iUxFMp4rbNQsmeftl3fZqnWU8i+B9P2frl8Pq6cpa40aR3KocfgH1Pl0E6oN4JAAAAABJRU5ErkJggg==);
```

Comment:
We should track an issue to use webpack for this. It would be great if this linked to an asset within the repo and automatically inlined it if it was below a certain size.